themes-book/opentextbook/functions.php

```diff
@@ -245,3 +245,17 @@ function pbt_explode_on_underscores( $string, $exclude = '' ) {
 
 	return $result;
 }
+
+/**
+ * Keep the "kitchen sink" open
+ * @param $in
+ * @return mixed
+ */
+function pressbooks_changeMceDefaults( $in ) {
```

Comment:
- function prefix should be `pbt_`
- function naming convention is snake_case
- bonus points for making it an anonymous function where the above two changes are moot.